src/azure-cli/azure/cli/command_modules/monitor/_help.py

```diff
@@ -1103,6 +1103,60 @@
     text: az monitor log-analytics workspace linked-service wait -n cluster -g MyResourceGroup --workspace-name MyWorkspace --created
 """
 
+helps['monitor log-analytics workspace linked-storage'] = """
+type: group
+short-summary: Manage linked storage account for log analytics workspace.
+"""
+
+helps['monitor log-analytics workspace linked-storage create'] = """
+type: command
+short-summary: Create linked storages for log analytics workspace.
+examples:
+  - name: Create a linked storage for a log analytics workspace
```

Comment:
>a [](start = 17, length = 1)

two

Reply:
I prefer using a instead of two. `linked-storage-account` is a resource which has several linked storage accounts. Two is quite strange.